src/components/textarea/textarea.style.ts

```diff
@@ -6,13 +6,15 @@ import { StyledLabelContainer } from "../../__internal__/label/label.style";
 import InputIconToggleStyle from "../../__internal__/input-icon-toggle/input-icon-toggle.style";
 import BaseTheme from "../../style/themes/base";
 
-export const MIN_HEIGHT = 64;
+export const DEFAULT_MIN_HEIGHT = 64;
 
 export interface StyledTextAreaProps {
   /** When true, label is placed in line an input */
   labelInline?: boolean;
   /** When true, adjusts padding for icon */
   hasIcon?: boolean;
+  /** Specify the mimimum height */
+  minHeight?: number;
```

Comment:
suggestion: rather than setting the prop type in two locations it would be better to extend the base Textarea with a Pick<>

You can see this being doing in other components, for example Card

Reply:
Refactor the code to include the above suggestion.